tests/test_python_install.sh

```diff
@@ -57,7 +57,11 @@ else # not virtualenv
         bin_name="python$python_mm"
     else
         macpie_bin="$PWD/pypy$python_mm-v$implementer_version-osx64/bin"
-        bin_name="pypy"
+        if [ "$(lex_ver $implementer_version)" -ge "$(lex_ver 7.3.2)" ]; then
+            bin_name="pypy3"
+        else
+            bin_name="pypy"
+        fi
```

Comment:
This was fixed in [the packaging](https://foss.heptapod.net/pypy/pypy/-/commit/bc470c4a1fe5c9e082842e7123369b8aa3866577) of pypy3.x for 7.3.3+, now it creates symlinks for `pypy` and others from `pypy3`. For python2.7 it is still only pypy

Reply:
nvrmind, this is in the test...